### ISSUE TITLE
Drawer icon shows when going back

### ISSUE THREAD

**Issue by Natriss** [dotnet/maui#23486]
### Description

There is no Drawer used but when Pushing a page and going back results in showing the Drawer icon.
It does disappear once the animation is finished.

https://github.com/dotnet/maui/assets/80769556/05d7a14d-bc66-4ced-8188-123e28faaa5f

### Steps to Reproduce

1. Create a .NET MAUI app
2. Make sure you have 2 pages
3. Add a button on page 1 that navigates to page 2 with Navigation.PushAsync()
4. Run the application
5. Go to page 2 and click on the back button

### Version with bug

8.0.3 GA

### Is this a regression from previous behavior?

Not sure, did not test other versions

### Last version that worked well

Unknown/Other

### Affected platforms

Android

**Comment by Natriss** [comment]
It's indeed a duplicate of https://github.com/dotnet/maui/issues/18867
My apologies.
